### ISSUE TITLE
multiple choices not supported?

### ISSUE THREAD

**Issue by olegsv** [ansman/validate.js#142]
Hello,

As far as we can see in 0.10 there is no support for SELECT with multiple="multiple" attribute. The library shows an error on the console.

We have a fix to get rid of the error, but it's far from complete solution:
...
```
@@ -554,9 +556,15 @@
       inputs = form.querySelectorAll("select[name]");
       for (i = 0; i < inputs.length; ++i) {
         input = inputs.item(i);
-        value = v.sanitizeFormValue(input.options[input.selectedIndex].value, options);
-        values[input.name] = value;
+        for( j=0; j<input.options.length; j++ ) {
+            select_option = input.options[j];
+            if( select_option.selected ) {
+                value = v.sanitizeFormValue( select_option.value, options);
+                values[input.name] = value ;
       }
+        }
+      }

```

**Comment by ansman** [comment]
Is this regarding `validate.collectFormValues()`?

**Comment by olegsv** [comment]
Yes this is.

**Comment by ansman** [comment]
Released in 0.11.0

**Comment by prionkor** [comment]
so, is there a way of validating multiselect? more precisely writing constraint for array values?
